sourced/ml/core/algorithms/token_parser.py

```diff
@@ -49,6 +52,14 @@ def __init__(self, stem_threshold=STEM_THRESHOLD, max_token_length=MAX_TOKEN_LEN
         self._single_shot = single_shot
         self._save_token_style = save_token_style
         self._attach_upper = attach_upper
+        if nn_model == "default":
```

Comment:
Let's not support "default". Just None or a UUID

Reply:
I agree that would be better but how to you precise you want to use the NN-based splitter rather than heuristic-based one, since we remove the "use_nn" parameter ?

Reply:
EDIT: ok